zebra/zebra_rib.c

```diff
@@ -1337,27 +1352,30 @@ static bool rib_update_re_from_ctx(struct route_entry *re,
 
 	/* If the new FIB set matches the existing FIB set, we're done. */
 	if (matched) {
-		if (IS_ZEBRA_DEBUG_RIB)
-			zlog_debug("%u:%s update_from_ctx(): existing fib nhg, no change",
-				   re->vrf_id, dest_str);
+		if (IS_ZEBRA_DEBUG_RIB) {
```

Comment:
we should keep no braces for short statements

Reply:
Fixed.

Reply:
not fixed :)

Reply:
oh, I don't know how I missed this. Now it should be ok.